CMakeLists.txt

```diff
@@ -30,6 +30,8 @@ include(${CMAKE_SOURCE_DIR}/src/js/CMakeLists.txt)
 option(libuv_buildtests "" OFF)
 add_subdirectory(deps/libuv EXCLUDE_FROM_ALL)
 add_subdirectory(deps/quickjs EXCLUDE_FROM_ALL)
+option(REPLXX_BuildExamples "" OFF)
```

Comment:
disable also building the dynamic library please, since we don't use it.

Reply:
Done!